buildSrc/src/main/groovy/swatch.spring-boot-dependencies-conventions.gradle

```diff
@@ -14,29 +14,10 @@ dependencies {
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine"
 }
 
-// Define all versions here so that all projects will use the same versions.
 dependencyManagement {
     imports {
         // See https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies to
         // get listings of the contents of this BOM
         mavenBom libraries["spring-boot-dependencies"]
```

Comment:
I'd be interested to go ahead and try moving this dep to the `dependencies` section, using `enforcedPlatform` and dropping the spring dependency management plugin; if it all works out, let's just go ahead and do that here.